doc/index.rst

```diff
@@ -39,10 +35,13 @@ Example
     task = openml.tasks.get_task(31)
     # Run the scikit-learn model on the task.
     run = openml.runs.run_model_on_task(clf, task)
-    # Publish the experiment on OpenML (optional, requires an API key).
+    # Publish the experiment on OpenML (optional, requires an API key.
```

Comment:
Should we also include in comments that they can use `openml.config.start_using_configuration_for_example()` for trials?
Unless this is to encourage signing in and creating an API.

Reply:
Signing up and using the own API key is definitively the better option. Also, this is the front page, so it should be as simple as possible.